apps/webapp/app/components/navigation/JobsMenu.tsx

```diff
@@ -18,6 +19,7 @@ export function JobsMenu({ matches }: { matches: RouteMatch[] }) {
   const [isOpen, setIsOpen] = useState(false);
   const organization = useOrganization(matches);
   const project = useProject(matches);
+  const projectJobs = useJobs();
```

Comment:
You should pass the `matches` into `useJobs()`. 

This is done because when switching between pages we were getting React errors about the number of hooks called changing. We call `useMatches` once in the `Breadcrumb` component, always. Then pass those matches to everything else. Seems to fix this issue…

Reply:
Good catch. I'll update the code with the suggested change. Thanks!